README.md

```diff
@@ -21,8 +21,52 @@ This is a multi module maven project relying on the [parent `pom.xml`](https://g
 
 * **client-skeleton-provider**: client skeleton module with the purpose of registrating a specific service into the Service Registry and running a web server where the service is available.
 
-Skeletons are built on the [`Arrowhead Client Library`](https://github.com/arrowhead-f/client-library-java-spring) which is also imported to this project as a maven dependency. The client library provides the `ArrowheadService.class` which is a singleton spring managed bean and designed with the purpose of interacting with Arrowhead Framework. Use its methods by [autowiring](https://www.baeldung.com/spring-autowire) into your spring managed custom classes or use `ArrowheadBeans.getArrowheadService()` if your custom class is not spring managed.
+Skeletons are built on the [`Arrowhead Client Library`](https://github.com/arrowhead-f/client-library-java-spring) which is also imported to this project as a maven dependency. The client library provides the `ArrowheadService.class` which is a singleton spring managed bean and designed with the purpose of interacting with Arrowhead Framework. Use its methods by [autowiring](https://www.baeldung.com/spring-autowire) into your spring managed custom classes or use `ArrowheadBeans.getArrowheadService()` if your custom class is not spring managed. *(**Look for the java docs** attached for each method within this class.)*
 
 Both client skeleton have a default 'ApplicationInitListener' and a defult 'SecurityConfig' what you can change or extend. The essential configuration has to be managed by customizing the `application.properties` file, located in `src/main/resources` folder.
 
+### Best practices to start with the skeletons
+
+##### (1st) apllicaction.properties
+Location: `src/main/resources`
+* Decide the required security level and set the `server.ssl.enabled` and `token.security.filter.enabled` properties accordingly.
+* Create your own client certificate (or for demo purpuse use the provided one) and update the further `server.ssl...` properties accordingly. *(**Note** that `server.ssl.key-store-password` and `server.ssl.key-password` must be the same.)*
+* Change the `client_system_name` property to your system name. *(**Note** that it should be in line with your certificate common name e.g.: when your certificate common name is `my_awesome_client.my_cloud.my_company.arrowhed.eu`, then your system name is  `my_awesome_client`)*
+* Adjust the Service Registry Core System location by the `sr_address` and `sr_port` properties.
+* In case of a provider you have to set it's web-server parameters by the `server.address` and `server.port` properties.
+* In case of a consumer decide whether it should act as a web-server or not. If yes, then set the `spring.main.web-application-type` to 'servlet' and set the further server parameters like in the provider case. If not, just left these properties untouched.
```

Comment:
* In case of a consumer decide**s** whether it should act as a web-server or not. If yes, then set the `spring.main.web-application-type` to 'servlet' and set ~the~ further server parameters like in the provider case. If not, just ~left~ **leave** these properties ~untouched~ **unchanged**.